datasets/cmu_mocap/cmu_mocap.py

```diff
@@ -0,0 +1,300 @@
+        "http://mocap.cs.cmu.edu/allmpg/allmpg_40to50-2.zip",
+        "http://mocap.cs.cmu.edu/allmpg/allmpg_40to50-3.zip",
+        "http://mocap.cs.cmu.edu/allmpg/allmpg_60to90.zip",
+    ],
+    "avi": "http://mocap.cs.cmu.edu/allavi.zip",
+}
+
+_METADATA_URL = "https://huggingface.co/datasets/dnaveenr/cmu_mocap/raw/main/subject_to_details.json"
+
+_CATEGORIES = {
+    "1": "Human Interaction",
+    "2": "Interaction with Environment",
+    "3": "Locomotion",
+    "4": "Physical Activities & Sports",
+    "5": "Situations & Scenarios",
+    "6": "Test Motions",
+    "": "",
+}
+
+_SUB_CATEGORY_NAMES = {
+    ("1", "1"): "two subjects",
+    ("2", "1"): "playground",
+    ("2", "2"): "uneven terrain",
+    ("2", "3"): "path with obstacles",
+    ("3", "1"): "running",
+    ("3", "2"): "walking",
+    ("3", "3"): "jumping",
+    ("3", "4"): "varied",
+    ("4", "0"): "golf",
+    ("4", "1"): "frisbee",
+    ("4", "2"): "dance",
+    ("4", "3"): "gymnastics",
+    ("4", "4"): "acrobatics",
+    ("4", "5"): "martial arts",
+    ("4", "6"): "racquet/paddle sports",
+    ("4", "7"): "soccer",
+    ("4", "8"): "boxing",
+    ("4", "9"): "general exercise and stretching",
+    ("5", "1"): "common behaviors and expressions",
+    ("5", "2"): "pantomime",
+    ("5", "3"): "communication gestures and signals",
+    ("5", "4"): "cross-category variety",
+    ("6", "1"): "",
+    ("", ""): "",
+}
+
+
+class CmuMocapConfig(datasets.BuilderConfig):
+    """BuilderConfig for CMU Mocap."""
+
+    def __init__(
+        self,
+        **kwargs,
+    ):
+        """BuilderConfig for MetaShift.
+
+        Args:
+            **kwargs: keyword arguments forwarded to super.
+        """
+        super(CmuMocapConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+
+
+class CmuMocap(datasets.GeneratorBasedBuilder):
+
+    BUILDER_CONFIGS = [
+        CmuMocapConfig(
+            name="asf-amc",
+            description="""\
+            The ASF file (Acclaim Skeleton File) is a skeleton file . AMC files (Acclaim Motion Capture data) are the motion files.
+            """,
+        ),
+        CmuMocapConfig(
+            name="c3d",
+            description="""\
+            The C3D format stores 3D coordinate information, analog data and associated information used in 3D motion data capture and subsequent analysis operations.
+            """,
+        ),
+        CmuMocapConfig(
+            name="mpg",
+            description="""\
+            The MPG format is a common video file that uses a digital video format standardized by the Moving Picture Experts Group (MPEG).
+          """,
+        ),
+        CmuMocapConfig(
+            name="avi",
+            description="""\
+            The AVI (Audio Video Interleave) format has been the long standing format to save and deliver movies and other video files.
+            """,
+        ),
+    ]
+
+    BUILDER_CONFIG_CLASS = CmuMocapConfig
+
```

Comment:
`asf-amc` was the initial format and has the most data, so let's make it the default config:
```suggestion
    BUILDER_CONFIG_CLASS = CmuMocapConfig
    DEFAULT_CONFIG_NAME = "asf-amc"
```